ipa-core/Cargo.toml

```diff
@@ -78,10 +81,10 @@ ipa-step-derive = { version = "*", path = "../ipa-step-derive" }
 aes = "0.8.3"
 async-trait = "0.1.79"
 async-scoped = { version = "0.9.0", features = ["use-tokio"], optional = true }
-axum = { version = "0.6", optional = true, features = ["http2", "macros"] }
+axum = { version = "0.7.5", optional = true, features = ["http2", "macros"] }
 # The following is a temporary version until we can stabilize the build on a higher version
 # of axum, rustls and the http stack.
-axum-server = { git = "https://github.com/cberkhoff/axum-server/", branch = "rustls-0.23", version = "0.5.3", optional = true, features = [
+axum-server = { git = "https://github.com/cberkhoff/axum-server/", branch = "0.6.1", version = "0.6.1", optional = true, features = [
```

Comment:
is 0.6.1 the right version for `axum` 0.7.5?

Reply:
0.6.0 is the "official" one. I had to patch it for this PR, hence the minor bump